x/nft/expected_keepers.go

```diff
@@ -16,4 +16,9 @@ type BankKeeper interface {
 type AccountKeeper interface {
 	GetModuleAddress(name string) sdk.AccAddress
 	GetAccount(ctx context.Context, addr sdk.AccAddress) sdk.AccountI
+
+	// StringToBytes decodes text to bytes
+	StringToBytes(text string) ([]byte, error)
+	// BytesToString encodes bytes to text
+	BytesToString(bz []byte) (string, error)
```

Comment:
Since x/nft already depends on core/address, would it be better to just embed `address.Codec` instead of copying the methods?

The godoc on these methods does not particularly help the reader, so I lean towards embedding the interface being more instructive as to purpose and intent.

Embedding the interface seems more congruous with the way this type is used in x/nft/keeper.NewKeeper, too.

Reply:
For consistency, best to modify it as well in the expected keepers having already these two methods.